CLI/test/ressources/dependencies_4/output_expected/error.txt

```diff
@@ -1,12 +0,0 @@
-﻿ReturnCode=ParsingDiagnostics_
```

Comment:
You have deleted this file because it is not relevant anymore, dependencie path is not correct so the test stops quickly. But why not comparing the console output as for the test like argument_errors_0..1..